src/snapred/backend/data/GroceryService.py

```diff
@@ -879,10 +886,8 @@ def fetchGroceryList(self, groceryList: List[GroceryListItem]) -> List[Workspace
                         item.version = self.dataService._getVersionFromCalibrationIndex(
                             item.runNumber, item.useLiteMode
                         )
-                    record = self.dataService.readCalibrationRecord(item.runNumber, item.useLiteMode, item.version)
-                    item.runNumber = record.runNumber
```

Comment:
I was on the verge of editing all of them to just take the item.  I think it was the diffcal output workspace name method that has around 5 or 6 arguments to it and I just got tired of writing it all out.  I had originally made them to use arguments to avoid forcing users to create grocery list items just to get a workspace or file name; but I don't think these methods are ever called outside of the grocery service anyway.